src/devices/Ssd1351/Command/Commands.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.Ssd1351.Command
+{
+    public enum SSD1351Command : byte
+    {
+        SSD1351_CMD_SETCOLUMN = 0x15, // used
+        SSD1351_CMD_SETROW = 0x75, // used
+        SSD1351_CMD_WRITERAM = 0x5C, //used
+        SSD1351_CMD_READRAM = 0x5D,
```

Comment:
I'd recommend renaming the values or alternatively making this enum private. If you choose renaming:
- drop prefix (`SSD1351_CMD_` seems redundand since it's already in the name of the enum)
- expand names where possible and makes sense. I.e.: `SetRow`, `DisplayOn`, `HorizontalScroll` etc